Resources/docs/introduction.rst

```diff
@@ -103,7 +103,7 @@ Place the following mapping file in the
     <massive-search-mapping xmlns="http://massive.io/schema/dic/massive-search-mapping">
 
         <mapping class="Model\Product">
-            <index name="product" />
+            <index value="product" />
```

Comment:
why is this here now value - i think "name" would be a better name for this property!